### ISSUE TITLE
[BUG]: NHL HITZ 2003 - Audio skipping/out-of-sync on intro video.

### ISSUE THREAD

**Issue by StillPlay2** [PCSX2/pcsx2#10063]
### Describe the Bug

The sound skips in places after the intro video starts.
Video provided below for more clarification.
I verified via Redump.org properties & came back as: Green tick: Verified as NHL Hitz 2003 (USA) [SLUS-20438] (Version 1.01).

### Reproduction Steps

As soon as the game starts.

### Expected Behavior

No skipping

### PCSX2 Revision

V1.7.5082

### Operating System

Windows 10 (64bit)

### CPU

AMD Ryzen 7 3800x

### GPU

AMD RX 6500XT

### GS Settings

Default setiings.

### Emulation Settings

Native.

### GS Window Screenshots

https://github.com/PCSX2/pcsx2/assets/116039192/d39b8867-a255-4dc1-85f3-3385c83f6230

**Comment by Buzzardsoul** [comment]
Can you untick I/O Processor (IOP, MIPS-1)'s Enable Recompiler and see if it helps?

**Comment by RedDevilus** [comment]
> As of 1.7.5901; audio still skips. Changing backend to Cubeb or SDL has no effect.

Well it doesn't look like an output problem, so changing settings in GUI will likely not do anything beneficial. From the video the audio samples are cut short for some reason.

**Comment by MrCK1** [comment]
After listening to the audio sample, I would say this is the same bug as https://github.com/PCSX2/pcsx2/issues/6419 (made by the same developer around a similar time period)

**Comment by Ziemas** [comment]
NFS at least did not seem to be an SPU bug, IIRC the game was just sending chunks of silent audio at it. The problem is probably elsewhere.

**Comment by mahoneyt944** [comment]
Had a similar issue with a different game, turned out I needed to change the output hz in Windows to match my tv. Mine was at 60, had to change it to 120. Might be worth a try.

Go to Settings > System > Display > Advanced display settings, select the display you want to change, then choose your desired refresh rate.

**Comment by mahoneyt944** [comment]
I can confirm that fixing the Windows settings in my previous post corrected all audio issues and now plays correctly using the 2.2 stable build.

**Comment by mahoneyt944** [comment]
That is odd. I'm using 2.2.0 stable build and the intro to NHL hitz 2003 does not skip for me.

**Comment by mahoneyt944** [comment]
Ntsc. I just recently built a PC specifically for pcsx2. So fresh install of pcsx2 on windows 11. When I first tried to play NHL hitz 2003 like you it skipped, after changing the hz in Windows to 120 and setting it to use freesync premium (as my Samsung TV uses), now it does not skip. This was also the case for burnout 3 when entering a race, it would skip every few seconds, but this was also fixed with that change in windows.

**Comment by mahoneyt944** [comment]
Here's a quick recording from my phone
https://youtu.be/oJxQLIetiLU?feature=shared

**Comment by mahoneyt944** [comment]
Ahh I think I found it. I'm using a 300% EE overclock. Changing it to 100 makes it skip

**Comment by mahoneyt944** [comment]
Yeah it looks like cycle rate of 130 or 180 it holds the fps around 30. Cycle rate 300 runs it up to 60 fps. When the cycle rate is 100 though it drops into the low 20s.

**Comment by mahoneyt944** [comment]
In GameDB, NHL Hitz 2002 is using `- EETimingHack`. Might need to test that with 2003 as well.

**Comment by ScottNash042** [comment]
still an issue using v2.7.42
